public-ui/src/app/data-browser/views/survey-view/components/survey-answer-react.component.tsx

```diff
@@ -234,9 +205,10 @@ const SurveyAnswerRowComponent = (class extends React.Component<SurveyRowProps,
             graphButtons.unshift('Survey Versions');
         }
         const participantPercentage = ((this.props.countValue / this.props.participantCount) * 100).toFixed(2);
+        const countString = countValue.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
         return <React.Fragment>
             <div className={drawerOpen ? 'active-row survey-tbl-exp-r survey-tbl-r' : 'survey-tbl-exp-r survey-tbl-r'}
-                onClick={() => this.openDrawer()}>
+                onClick={() => this.openDrawer(answerValueString)}>
```

Comment:
Sending this props to not open the drawer for 'did not answer' since it does not have a chart.